packages/ui-core/shared/src/lib/table-components/editors/employee-link-editor.component.ts

```
@@ -0,0 +1,32 @@
+			</a>
+		</ng-container>
+	`,
+	styleUrls: ['../employee-links/employee-links.component.scss']
+})
+export class EmployeeLinkEditorComponent extends DefaultEditor implements OnInit {
+	@Input() cell!: Cell; // Input to access the cell data
+	value!: IUser;
+
+	constructor() {
+		super();
+	}
```

Biome

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)